src/main/java/org/tugraz/sysds/runtime/instructions/fed/AggregateBinaryFEDInstruction.java

```diff
@@ -56,7 +61,7 @@ public static AggregateBinaryFEDInstruction parseInstruction(String str) {
 		String[] parts = InstructionUtils.getInstructionPartsWithValueType(str);
 		String opcode = parts[0];
 		
-		if( !opcode.equalsIgnoreCase("ba+*") ) {
+		if (!opcode.equalsIgnoreCase("ba+*")) {
```

Comment:
Import the codestyle

Reply:
I will not apply the code style for the whole file yet, so whoever looks at the PR can clearly see the changes. Whoever merges can apply it.